starlite/__main__.py

```diff
@@ -0,0 +1,3 @@
+from starlite.cli.main import starlite_group  # pragma: no cover
```

Comment:
can you move this file into the cli module? does it need to be in the starlite root level?

Reply:
Doesn't need to be, but this makes `python -m starlite` possible, which could be advantageous in some situations. Putting this into `__main__.py` is the canonical way to invoke a package with a CLI so I thought it's good if we adhere to this standard. I don't see any downsides to this. Wdyt?